python/lsst/pipe/base/config.py

```diff
@@ -200,6 +200,7 @@ class to allow configuration of the connections class. This dynamically
         default=True,
         optional=False,
         doc="Flag to enable/disable metadata saving for a task, enabled by default.",
+        deprecated="This field is deprecated and will be removed after v25.",
```

Comment:
I think it has to b v26 because v25 is already being worked on from earlier in the year.